decidim-blogs/app/commands/decidim/blogs/admin/create_post.rb

```diff
@@ -28,12 +28,11 @@ def call
         private
 
         def create_post!
-          attributes = {
+          attributes = {º
```

Comment:
is that character intended ?

Reply:
thanks for the warning, should be solved now